### PR TITLE
docs: update reload instructions (#49379)Co-authored-by: Oliver Eyton-Williams <ojeytonwilliams@gmail.com> Co-authored-by: Naomi Carrigan <nhcarrigan@gmail.com>

### PR DESCRIPTION
**Pull request by raisedadead**
Because @ojeytonwilliams is evil and doesn't update docs.

### CODE REVIEW

File: docs/devops.md [hunk 1]
```diff
@@ -534,13 +534,13 @@ npm run create:config && npm run build:curriculum && npm run build:server
 4. Start Instances
 
 ```console
-pm2 start all --update-env && pm2 logs
-```
+   cd api-server && pm2 reload ecosystem.config.js cd .. && pm2 logs
```
Comment (ojeytonwilliams) [inline docs/devops.md:537]:
```suggestion
   cd api-server && pm2 start ecosystem.config.js && cd .. && pm2 logs
```

File: docs/devops.md [hunk 2]
```diff
@@ -534,13 +534,13 @@ npm run create:config && npm run build:curriculum && npm run build:server
...
+   ```
 
 #### 2. Rolling updates - Used for logical changes to code.
 
 ```console
-pm2 reload all --update-env && pm2 logs
+cd api-server && pm2 reload ecosystem.config.js cd .. && pm2 logs
```
Comment (ojeytonwilliams) [inline docs/devops.md:543]:
```suggestion
cd api-server && pm2 reload ecosystem.config.js && cd .. && pm2 logs
```

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]

**Comment by ojeytonwilliams** [comment]
Ah, but who reviewed the change? :thinking:

File: docs/devops.md (hunk 1 above)
Comment (raisedadead) [reply docs/devops.md:537]: [Do you even look at your own changes?](https://github.com/freeCodeCamp/freeCodeCamp/pull/49379/files#diff-b4e944b268ad7c01bdcd9d3e05f76e12718cbd446e8ed55447145b92daca61d6L488-L494)

Comment (ojeytonwilliams) [reply docs/devops.md:537]: Does it matter? The main issue was the missing `&&`.

Comment (raisedadead) [reply docs/devops.md:537]: 🤦🏽‍♂️

File: docs/devops.md [hunk 3]
```diff
@@ -534,13 +534,13 @@ npm run create:config && npm run build:curriculum && npm run build:server
 4. Start Instances
 
 ```console
-pm2 start all --update-env && pm2 logs
-```
+   cd api-server && pm2 start ecosystem.config.js && cd .. && pm2 logs
```
Comment (naomi-lgbt) [inline docs/devops.md:537]:
```suggestion
cd api-server && pm2 start ecosystem.config.js && cd .. && pm2 logs
```

Extra spacing?

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

**Review by naomi-lgbt — APPROVED** [review]

**Review by raisedadead — COMMENTED** [review]
Because Oliver is evil.

**Pull request merged**